### PR TITLE
update CDSi supporting data to v4.31, primary series names and unit t…

### PR DESCRIPTION
**Pull request by nkarip**
…ests

### CODE REVIEW

**Review by karbates — APPROVED** [review]
I reviewed the changes and do not see any issues

File: src/components/Validator/Validator.test.js [hunk 1]
```diff
@@ -414,3 +414,200 @@ test('2022-0022: Dose #1 is AstraZeneca Dose #2 is Sinovac Covid-19 and Vaccine
...
+test('2022-0030: Patient is 12 years of age and has been administered CVX 221 of the Moderna Covid-19 vaccine as the first dose', () => {
+  const patientBundle = createPatientBundle(new Date(2010, 5, 27), [
+    { dateAdministered: '2022-06-27', cvx: '221' }
+  ]);
+  const values = Validator.execute(patientBundle, vcTypes);
+  expect(validPrimarySeries(values)).toBe(false);
+});
+
+test('2022-0031: Patient is 12 years of age and has been administered Moderna\'s (CVX 221) Covid-19 vaccine for the first and second dose', () => {
+  const patientBundle = createPatientBundle(new Date(2020, 4, 30), [
```
Comment (sarahmcdougall) [inline src/components/Validator/Validator.test.js:493]:
```suggestion
  const patientBundle = createPatientBundle(new Date(2010, 4, 30), [
```

File: src/components/Validator/Validator.test.js [hunk 2]
```diff
@@ -414,3 +414,200 @@ test('2022-0022: Dose #1 is AstraZeneca Dose #2 is Sinovac Covid-19 and Vaccine
...
+    { dateAdministered: '2022-05-30', cvx: '221' },
+    { dateAdministered: '2022-06-27', cvx: '221' }
+  ]);
+  const values = Validator.execute(patientBundle, vcTypes);
+  expect(validPrimarySeries(values)).toBe(true);
+});
+
+test('2022-0032: Patient is 15 months of age and has been administered the first dose of the Moderna Covid-19 (CVX 228) vaccine', () => {
+  const patientBundle = createPatientBundle(new Date(2021, 2, 27), [
+    { dateAdministered: '2022-06-27', cvx: '218' }
```
Comment (sarahmcdougall) [inline src/components/Validator/Validator.test.js:503]:
```suggestion
    { dateAdministered: '2022-06-27', cvx: '228' }
```

File: src/components/Validator/Validator.test.js [hunk 3]
```diff
@@ -414,3 +414,200 @@ test('2022-0022: Dose #1 is AstraZeneca Dose #2 is Sinovac Covid-19 and Vaccine
...
+  ]);
+  const values = Validator.execute(patientBundle, vcTypes);
+  expect(validPrimarySeries(values)).toBe(false);
+});
+
+test('2022-0033: Patient is 15 months of age and has been administered the second dose of the Moderna Covid-19 (CVX 228) vaccine', () => {
+  const patientBundle = createPatientBundle(new Date(2021, 1, 28), [
+    { dateAdministered: '2022-05-28', cvx: '218' },
+    { dateAdministered: '2022-06-25', cvx: '218' }
```
Comment (sarahmcdougall) [inline src/components/Validator/Validator.test.js:512]:
```suggestion
    { dateAdministered: '2022-05-28', cvx: '228' },
    { dateAdministered: '2022-06-25', cvx: '228' }
```

File: src/components/Validator/Validator.test.js [hunk 4]
```diff
@@ -414,3 +414,200 @@ test('2022-0022: Dose #1 is AstraZeneca Dose #2 is Sinovac Covid-19 and Vaccine
...
+  ]);
+  const values = Validator.execute(patientBundle, vcTypes);
+  expect(validPrimarySeries(values)).toBe(true);
+});
+
+test('2022-0034: Patient is 2 years of age and has been administered the Pfizer Covid-19 vaccine as a first dose and Moderna as second dose for the Covid-19 vaccine', () => {
+  const patientBundle = createPatientBundle(new Date(2020, 5, 30), [
```
Comment (sarahmcdougall) [inline src/components/Validator/Validator.test.js:519]:
```suggestion
  const patientBundle = createPatientBundle(new Date(2020, 4, 30), [
```

**Review by sarahmcdougall — CHANGES_REQUESTED** [review]

**Review by sarahmcdougall — APPROVED** [review]

**Pull request merged**